.github/workflows/build.yml

```diff
@@ -21,29 +21,37 @@ jobs:
       matrix:
         image_flavor: [main, nvidia]
         image_name: [silverblue, kinoite, vauxite, sericea, base, lxqt, mate]
-        major_version: [37, 38]
+        major_version: [37, 38, 39]
         nvidia_version: [0, 470, 535]
         include:
           - major_version: 37
             is_latest_version: false
             is_stable_version: true
+            is_gts_version: false
           - major_version: 38
             is_latest_version: true
             is_stable_version: true
+            is_gts_version: true
+          - major_version: 39
+            is_latest_version: true
```

Comment:
I can't remember the decision - are we marking beta releases as the latest?  

I seem to think we wanted latest to be the latest stable release

Reply:
Latest and stable are required to push to the latest tag so the latest stable release is what gets pushed there. 39 won't get pushed to that until it is marked as stable. When that happens, we set the flag for latest and stable releases to false for 38 but keep gts true and that will be the release that stays behind. Setting the latest flag here for 39 won't have an impact on our tags as it isn't marked stable yet. I know that sounds confusing too

Reply:
Thanks!  Happy for this to be merged then

Reply:
Awesome! Let's get this rolled out :)

Reply:
Nice work y'all!!